src/debugmode.lua

```diff
@@ -66,7 +66,9 @@ function Debug:draw()
 	end
 
 	local chartWidth = #self.drawTimeLogger.times
-	for i, t in ipairs(self.drawTimeLogger.times) do
+	local times = self.drawTimeLogger.times
+	for i = chartWidth, 1, -1 do
+		local t = times[i]
 		local y = math.floor(love.graphics.getHeight() - t*1000)
 		love.graphics.points(i, y)
 	end
```

Comment:
I decided to reverse the order of times in the table. It makes more sense in my brain that `times[0]` means "zero frames ago" and `times[10]` means "10 frames ago". So now we iterate over the table backwards. The newest frame still gets drawn on the right side of the chart (positive time is to the right).

Previously, `times[0]` was the oldest frame and `times[300]` was the newest frame.

Reply:
Or another way to do this would be to iterate through the table in forward order, but draw in reverse order (`chartWidth-x`):

```lua
local chartWidth = #self.drawTimeLogger.times
for x, t in ipairs(self.drawTimeLogger.times) do
  local y = math.floor(love.graphics.getHeight() - t*1000)
  love.graphics.points(chartWidth-x, y)
end
```